modules/map/template.html

```diff
@@ -0,0 +1,23 @@
+<section class="snowball-block snowball-block-text">
+	
+	<iframe src="{{embedUrlTarget}}" frameborder="0" width="100%" height="80%"></iframe>
```

Comment:
What happens when height is set to 80%? Might want to give it a standard value like 400px?